client/source/index.jsx

```diff
@@ -1,22 +1,32 @@
 import React from 'react';
 import ReactDOM from 'react-dom';
 import $ from 'jquery';
+import axios from 'axios';
 
 
 class App extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      updateData: []
     }
 
   }
 
+  componentDidMount() {
+    var newId = window.location.href;
+    newId = newId.substring(newId.length - 3, newId.length-1);
```

Comment:
i'd recommend setting this to a different variable instead of mutating the newId variable